EStimLibrary.sln

```diff
@@ -5,23 +5,26 @@ VisualStudioVersion = 17.0.31903.59
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{7E35BD38-1080-4A2E-B8B4-7CB0C5C08607}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EStimLibrary", "src\EStimLibrary\EStimLibrary.csproj", "{23710686-EDC8-41FC-AAC8-52B002F488D6}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EStimLibrary", "src\EStimLibrary\EStimLibrary.csproj", "{23710686-EDC8-41FC-AAC8-52B002F488D6}"
```

Comment:
Revert changes to the EStimLibrary.sln file. You should be able to use the tests/EStimLibrary.UnitTests/EStimLibrary.UnitTests.sln to open both the library and unit test projects at once. EStimLibrary.sln is just for the library